src/units.cpp

```diff
@@ -632,7 +632,7 @@ bool Units::requiresImports() const
         for (size_t u = 0; u < unitCount(); ++u) {
             unitAttributes(u, ref, prefix, exponent, multiplier, id);
             auto child = model->units(ref);
-            if (child == nullptr) {
+            if (child == nullptr || this == child.get()) {
```

Comment:
Parentheses: `if ((child == nullptr) || (this == child.get())) { `.

Reply:
[2b8c06d](https://github.com/cellml/libcellml/pull/1145/commits/2b8c06d41a5400303f6171ed91d5d69ff035bcba)